packages/block-editor/src/hooks/anchor.js

```diff
@@ -71,7 +71,18 @@ export const withInspectorControl = createHigherOrderComponent( ( BlockEdit ) =>
 					<InspectorAdvancedControls>
 						<TextControl
 							label={ __( 'HTML Anchor' ) }
-							help={ __( 'Anchors lets you link directly to a section on a page.' ) }
+							help={ (
+								<>
+									<p>
+										{ __( 'Enter a word or two — without spaces — to make a unique web address just for this heading, called an "anchor." Then, you\'ll be able link directly to this section of your page.' ) }
+									</p>
+									<p className="components-base-control__learn-more">
```

Comment:
It's a little strange to see this class name here since it's defined in a different package. 

Having said that, I can't think of any reason why it's a problem since `@wordpress/block-editor` has a dependency on `@wordpress/components`, so 🤐!

Reply:
Is there a better approach? I admit I don't know what I'm doing 😅

Reply:
Looking at this again, it doesn't look like this class name is doing anything. There's no CSS definition anywhere for `components-base-control__learn-more`.

```suggestion
									<p>
```